### PR TITLE
feat: add additional parameters for QR code generator app

### LINKED ISSUE #9157: UTM parameter for QR code generator
### Is your proposal related to a problem?

The link in the web interface or the QR Code generator is the same, resulting in no detailed insights when using analytics software like Fathom. I cannot tell which way of linking works best.

### Describe the solution you'd like

For analytics purposes I would love to add an UTM parameter to the URL so I can understand much better which way works better (or best).
For the start it would be sufficient to just have a fixed UTM parameter for each QR Code.

### Describe alternatives you've considered

Generating a QR code with a third party tool let me add an UTM parameter by hand to the URL.

### PR DESCRIPTION
**Pull request by G3root**
## What does this PR do?

Fixes #9157 

[Screencast 2023-05-29 16:05:00.webm](https://github.com/calcom/cal.com/assets/84864519/36975b48-dd5b-4bd0-8c38-91387f3a9c87)

## Type of change

- New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by hariombalhara** [comment]
The URL should be ?abc=1. ?should be prefixed automatically.
Screenshot 2023-06-01 at 3 00 54 PM

File: packages/app-store/qr_code/components/EventTypeAppCardInterface.tsx [hunk 1]
```diff
@@ -3,23 +3,27 @@ import { useState } from "react";
...
 import type { appDataSchema } from "../zod";
 
 const EventTypeAppCard: EventTypeAppCardComponent = function EventTypeAppCard({ eventType, app }) {
+  const { t } = useLocale();
   const [getAppData, setAppData] = useAppContextWithSchema<typeof appDataSchema>();
   const [enabled, setEnabled] = useState(getAppData("enabled"));
+  const [additionalParameters, setAdditionalParameters] = useState("");
 
-  const eventTypeURL = eventType.URL;
+  const url = additionalParameters !== "" ? `&${additionalParameters}` : "";
```
Comment (hariombalhara) [inline packages/app-store/qr_code/components/EventTypeAppCardInterface.tsx:17]:
```suggestion
  const query = additionalParameters !== "" ? `&${additionalParameters}` : "";
```

**Review by hariombalhara — CHANGES_REQUESTED** [review]
Left a comment.

**Comment by G3root** [comment]
@hariombalhara pushed requested changes

**Review by hariombalhara — APPROVED** [review]

**Pull request merged**